packages/client/src/api/ApiClient/ApiClient.ts

```diff
@@ -76,4 +76,10 @@ export const ApiClient: IApiClient = {
   async createMessage(body) {
     return ApiForumService.post(`/message`, body);
   },
+  async likeMessage(id: number, userId: string) {
+    return ApiForumService.post(`/message/like/${id}`, { userId: userId });
+  },
+  async dislikeMessage(id: number, userId: string) {
```

Comment:
Не критично, но имхо лучше каждый ID обзывать по-своему. Тогда чуть меньше времени уходит на чтение и восприятие кода )
```suggestion
  async likeMessage(messageId: number, userId: string) {
    return ApiForumService.post(`/message/like/${id}`, { userId: userId });
  },
  async dislikeMessage(messageId: number, userId: string) {
```